src/components/books/Books.js

```diff
@@ -4,9 +4,11 @@ import { fetchBooks, removeAllBooks } from '../../redux/books/books';
 import Book from '../book/Book';
 import AddNewBook from '../form/AddNewBook';
 
+// Books Component
 function Books() {
   const dispatch = useDispatch();
 
+  // fetch books from the store
   useEffect(() => {
     dispatch(fetchBooks());
```

Comment:
Please make sure all the books are shown even if the page is refreshed and books are updated. It means that they should come exactly from the API and get stored in the redux store.